guestbook/README.md

```diff
@@ -4,704 +4,294 @@
-  # selector:
-  #   matchLabels:
-  #     app: guestbook
-  #     role: master
-  #     tier: backend
-  template:
-    metadata:
-      labels:
-        app: redis
-        role: master
-        tier: backend
-    spec:
-      containers:
-      - name: master
-        image: gcr.io/google_containers/redis:e2e
-        resources:
-          requests:
-            cpu: 100m
-            memory: 100Mi
-        ports:
-        - containerPort: 6379
-```
+        kubectl get pods
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/guestbook/redis-master-deployment.yaml)
-<!-- END MUNGE: EXAMPLE redis-master-deployment.yaml -->
+   The response should be similar to this:
 
-#### Define a Service
+        NAME                            READY     STATUS    RESTARTS   AGE
+        redis-master-1068406935-3lswp   1/1       Running   0          28s
 
-A Kubernetes [Service](http://kubernetes.io/docs/user-guide/services/) is a named load balancer that proxies traffic to one or more containers. This is done using the [labels](http://kubernetes.io/docs/user-guide/labels/) metadata that we defined in the `redis-master` pod above.  As mentioned, we have only one redis master, but we nevertheless want to create a Service for it. Why? Because it gives us a deterministic way to route to the single master using an elastic IP.
 
-Services find the pods to load balance based on the pods' labels.
-The selector field of the Service description determines which pods will receive the traffic sent to the Service, and the `port` and `targetPort` information defines what port the Service proxy will run at.
+4. Run the following command to view the logs from the Redis Master Pod:
 
-The file [redis-master-service.yaml](https://git.k8s.io/examples/guestbook/redis-master-service.yaml) defines the redis master Service:
+        kubectl logs -f POD-NAME
 
-<!-- BEGIN MUNGE: EXAMPLE redis-master-service.yaml -->
+**Note:** Replace POD-NAME with the name of your Pod.
+{: .note}
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: redis-master
-  labels:
-    app: redis
-    role: master
-    tier: backend
-spec:
-  ports:
-    # the port that this service should serve on
-  - port: 6379
-    targetPort: 6379
-  selector:
-    app: redis
-    role: master
-    tier: backend
-```
+### Creating the Redis Master Service
+The guestbook applications needs to communicate to the Redis master to write its data. You need to apply a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) to proxy the traffic to the Redis master Pod. A Service defines a set of Pods and a policy to access them.
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/guestbook/redis-master-service.yaml)
-<!-- END MUNGE: EXAMPLE redis-master-service.yaml -->
+1. Apply the Redis Master Service from the following `redis-master-service.yaml` file: 
 
-#### Create a Service
+        kubectl apply -f redis-master-service.yaml
```

Comment:
you have too much space, the rendered code block has 4 spaces at the beginning, throughout the doc. Please start all code block lines at 4 spaces.